c8run/main.go

```diff
@@ -1,23 +1,20 @@
 package main
 
 import (
+	"c8run/internal/unix"
+	"c8run/internal/windows"
 	"errors"
+        pkgErrors "github.com/pkg/errors"
```

Comment:
This package is archived since 2021.

Reply:
Do you have recommendations for a replacement for this library? I kinda introduced the pkg/errors module as result of another review comment.

Reply:
I removed the dependency on pkg/errors and used a combination of stdlib errors package and runtime/debug to produce stacktraces.